src/boardgameprojectfile/boardgameprojectfile.cpp

```diff
@@ -0,0 +1,55 @@
+#include "boardgameprojectfile.h"
+#include <QFile>
+#include <QDataStream>
+#include <QString>
+#include <QDebug>
+
+BoardGameProjectFile::BoardGameProjectFile()

```

Comment:
Why you need constructor there? Just remove it at all or make it default in header 'BoardGameProjectFile() = default'